src/components/AppInfoConfigOption.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SelectInLine } from '@newrelic/gatsby-theme-newrelic';
+import MDXContainer from './MDXContainer';
+
+const AppInfoConfig = ({
+  optionType,
+  selectOptions,
+  pageState,
+  setPageState,
+  mdx,
+}) => {
+  const select = selectOptions.find(
+    (select) => select.optionType === optionType
+  );
+  return (
+    <div>
+      <SelectInLine
+        key={select.label}
+        label={select.label}
+        onChange={(e) =>
+          setPageState({
+            ...pageState,
+            selectOptions: {
+              ...pageState.selectOptions,
+              [select.label]: e.target.value,
+            },
+          })
+        }
+      >
+        {select.options.map((option) => (
+          <option key={option.label} value={option.value}>
+            {option.displayName}
+          </option>
+        ))}
+      </SelectInLine>
+
+      {mdx && <MDXContainer body={mdx} />}
```

Comment:
would we be able to destructure the `body` property off of mdx, because likely we will pass in the entire object, so it's really `mdx.body` that would get passed down to the `MDXContainer`

Reply:
yep!